fe/fe-core/src/main/java/org/apache/doris/clone/DynamicPartitionScheduler.java

```diff
@@ -63,6 +63,8 @@
 import java.util.Map;
 import java.util.Set;
 
+import static org.apache.doris.catalog.DynamicPartitionProperty.NOT_SET_HISTORY_PARTITION_NUM;
```

Comment:
There is no need to import the static value. You can use it directly. `DynamicPartitionProperty.NOT_SET_HISTORY_PARTITION_NUM`